code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5278,6 +5278,28 @@
 	// it also makes you hungry because it speeds up your metabolism
 	M.nutrition--
 
+/datum/reagent/tendies
+	name = "Tendies"
+	id = TENDIES
+	description = "Gimme gimme chicken tendies, be they crispy or from Wendys."
+	nutriment_factor = REAGENTS_METABOLISM
+	color = "#AB6F0E" //rgb: 171, 111, 14
+	density = 5
+	specheatcap = 1
+
+/datum/reagent/tendies/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+
+	M.nutrition += REM * nutriment_factor
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.job == "Janitor")
```

Comment:
You want that to be `H.mind.assigned_role`.

Reply:
Fairly sure this would mean this effect wouldn't work on cloned Jannies.

Reply:
Please update sprinkles to this system too while you're at it